openupgrade_scripts/scripts/fleet/14.0.0.1/post-migration.py

```diff
@@ -87,3 +150,13 @@ def migrate(env, version):
         ],
     )
     recompute_fleet_vehicle_log_contract_name(env)
+    
+    # Move data from fleet_vehicle_cost and fleet_vehicle_log_fuel to fleet_vehicle_log_services
+    other_service_type = None
+    fuel_service_type = None
+    if _check_vehicle_fuel_cost_have_no_type(env):
+        fuel_service_type = _generate_fuel_service_type(env)
+    if _check_vehicle_cost_have_no_type(env):
+        other_service_type = _generate_other_service_type(env)
+    _move_data_from_cost_to_service(env, other_service_type)
+    _move_data_from_fuel_to_service(env, fuel_service_type)
```

Comment:
thiếu đoạn code để set state của module `viin_fleet` thành `to install`

Reply:
vâng, em chỉnh lại rồi ạ